NoodleBox/source/sequence_layer.h

```diff
@@ -1025,6 +1026,10 @@ class CSequenceLayer {
 			// work out the next "grid" step position
 			clock::TICKS_TYPE next_step_grid_time = ticks_per_step * (int)(1.5+(double)ticks/ticks_per_step);
```

Comment:
I'm a bit unsure how this line works. Why isn't this computed using only integer math on ticks?